### DISCUSSION TITLE
Component card data & layout

### DISCUSSION THREAD

**Discussion by lencodes** [OpenCatalogi/.github#187] [Ideas]
Component cards komen terug op meerdere plekken (o.a. componenten zoekpagina, component detailpagina). Ze zijn belangrijke vertoningen van componenten en hebben effect op hoe gebruikers componenten kunnen zoeken en vinden.

Wij stellen nu voor om de volgende gegevens in een component card te tonen:

1. Titel
2. Description
3. Laag
4. Categorie (gebaseerd op [deze kaart](https://www.gemmaonline.nl/images/gemmaonline/a/a0/GemeentelijkGegevenslandschap.png); incl. icoon per categorie)
5. Status
6. Aantal installaties
7. Organisatie
8. Type licentie 
9. (Optionele) link naar GitHub.

Deze data willen op de volgende manier vormgeven:

![Cards@2x](https://user-images.githubusercontent.com/66552932/184122519-636e6675-ce53-44e5-9617-b9a8f3b1a778.jpg)

**Comment by RonaldvCortenberghe** [comment]
akkoord met voorstel
